tests/ha/test_server_pod_failure.py

```diff
@@ -558,13 +519,12 @@ def test_writes_during_server_pod_down(self):
                                                     log_prefix=self.test_prefix, skipcleanup=True,
                                                     nsamples=2, nclients=2)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 7: Successfully created IAM user with multiple buckets and ran IOs "
-                    "when cluster has some failures due to server pod %s has gone down.", pod_name)
+        LOGGER.info("Step 4: Successfully created IAM user with multiple buckets and ran IOs.",)
```

Comment:
when server pod was down.

Reply:
initial statement has already mentioned the same. Not required here.